velox/exec/HashJoinBridge.cpp

```diff
@@ -33,6 +66,123 @@ void HashJoinBridge::addBuilder() {
   ++numBuilders_;
 }
 
+namespace {
+// Spills the row container from one of the sub-table from
+// 'table' to parallelize the table spilling. The function
+// spills all the rows from the row container and returns the spiller for the
+// caller to collect the spilled partitions and stats.
+std::unique_ptr<Spiller> createSpiller(
+    RowContainer* subTableRows,
+    HashBitRange hashBitRange,
+    const std::shared_ptr<const core::HashJoinNode> joinNode,
+    const common::SpillConfig* spillConfig,
+    folly::Synchronized<common::SpillStats>* stats) {
+  VELOX_CHECK_NOT_NULL(joinNode);
+  const auto joinType = joinNode->joinType();
+  const auto tableType = joinTableType(joinNode);
```

Comment:
Let's pass joinType and tableType to createSpiller so we don't need to create joinTableType multiple times.